### DISCUSSION TITLE
Schemas for simple synthesis processes

### DISCUSSION THREAD

**Discussion by budschi** [FAIRmat-NFDI/AreaA-data_modeling_and_schemas#88] [Task Force Meeting]
Agenda: we want to define a list of schemas for basic synthesis processes in our plugin NOMAD Materials Processing (E.g. sample cutting, make a solution, etc.). In this TF meeting we would like to collect your needs and plan this task.

* weighing/scaling pipetting, dosage (flow rate)
* "simple" characterization: lenght, weight, 
* sieving
* grinding / milling

* adjusting a pH value, titration
* calcination
* washing, rinsing
* centrifugation
* rotary evaporation
* filtration, ultrafiltration
* heating, cooking, bath
* all the used inventory: beaker, flask, open or closed, atmosphere

* sample cutting: cleaving, dicing, sawing, slicing, --> Andrea
* Surface treatments: surface activation, surface cleaning, surface etching, surface polishing, --> Tamas
* cleaning methods: ultra-sonic bath, brushing, 
* drying methods: freeeze, blow, evaporating, N2 blow drying, critical-point drying (low and high T) --> Fabian

* solution preparation --> Tamas
  * scaling
  * pipetting
  * heating up
  * stirring, shaking

* powder preparation --> Andrea

What should be done:
* yaml file
* write a structure
* if possible provide link to an ontology (CHMO)
* do something unitl mid January
*

**Comment by andreaa93** [comment]
interesting,

 I already need some "solution preparation" class for the MOVPE use case
